ui/src/components/SituationCard.vue

```diff
@@ -100,7 +102,7 @@ const handleSituationSelected = () => {
 .title-row {
 	display: flex;
 	justify-content: space-between;
-	margin-bottom: 15px;
+	margin-bottom: 10px;
```

Comment:
I don't think it's required, this is just me: I try to use `spacing` from FeatherDS variables to keep space consistency of the whole app, when I can.